testcases/semantics/json/package_parameter_keyword.json

```diff
@@ -0,0 +1,86 @@
+{
+  "modules": [],
+  "packages": [
+    {
+      "identifier": "TestPackage",
+      "parameters": [
+        {
+          "identifier": "a",
+          "expression": "8'sb10101010",
+          "paramtype": "Parameter",
```

Comment:
This looks like a bug to me.
In a package, the keyword `parameter` is simply an alias for `localparam`.
So all parameters in a package should have `"paramtype": "Parameter"`.

Reply:
Fixed.